docs/source/tools/navigator/database.rst

```diff
@@ -1,6 +1,11 @@
 .. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
+.. NOTE: This page is currently not included in the docs since
```

Comment:
The docs build won't like that. Could you include it as a direct sub-page of the navigator, or as a stand-alone page in early access?

Reply:
Ah crap, I thought not including it in any toc tree means it just won’t be build but apparently it’s still built and we get a warning. I don’t really want to include it anywhere. It was a subpage of navigator console before, we never documented that you could use it outside of Navigator console. I’ll just kill it completely, we can always resurrect it from git history if we do but I expect that even if we make database more prominent for perf reasons, the database will be an implementation detail.